src/main/scala/io/iohk/ethereum/vmrunner/State.scala

```diff
@@ -25,7 +25,7 @@ object State {
     val tx = MockVmInput.transaction(creatorAddress, code, balance, gas)
     val bh = MockVmInput.blockHeader
 
-    val context: PC = ProgramContext(tx, bh, world, evmConfig)
+    val context: PC = ProgramContext(tx, newAddress, Program(code), bh, world, evmConfig)
```

Comment:
Minor detail: Should we do here what is now done on `prepareProgramContext` and was previously done when creating the `ProgramContext`? I'm wondering if there could be a problem when using the `evm-runner` now due to `newAddress` not being saved to the world (and also the value not being transfered)